### PR TITLE
Dipole

### PR DESCRIPTION
**Pull request by davkovacs**

### CODE REVIEW

File: src/electrostatics.jl [hunk 1]
```diff
@@ -0,0 +1,87 @@
+"""
+`module Electro`
+
+This module implements various functions related to (long range) electrostatic interactions
+"""
+
+module Electro
```
Comment (cortner) [inline src/electrostatics.jl:7]: would it be ok to rename it `Electrostatics`

File: test/test_electro.jl [hunk 2]
```diff
@@ -0,0 +1,40 @@
+@testset "electrostatics.jl" begin
```
Comment (cortner) [inline test/test_electro.jl:1]: The string is commonly just a description, doesn't need to match the filename. I'm don't particularly care though...

**Review by cortner — APPROVED** [review]
look good, more tests, and in particular putting this into the models can come over time?

File: src/electrostatics.jl (hunk 1 above)
Comment (davkovacs) [reply src/electrostatics.jl:7]: Of course, I am happy to rename it.

**Pull request merged**